changelog/add-croatia-fees

```diff
@@ -0,0 +1,4 @@
+Significance: major
+Type: add
+
+Add support for Croatia fees
```

Comment:
I think we can exclude the word "fees" here.

Reply:
changed. thx